pennylane_lightning_gpu/src/bindings/Bindings.cpp

```diff
@@ -26,6 +26,10 @@
 #include "DevTag.hpp"
 #include "DevicePool.hpp"
 #include "Error.hpp"
+#ifdef ENABLE_MPI
+#include "MPIManager.hpp"
+#include "StateVectorCudaMPI.hpp"
+#endif
```

Comment:
Could you please merge the `#ifdef ENABLE_MPI` blocks into a single one? This helps to quickly understand what is being added when MPI is enabled. I mean, at least the blocks in the header part (the first two).

Reply:
done!